nrniv/Parallel/alltoall_dict.py

```diff
@@ -5,16 +5,16 @@
 rank = int(pc.id())
 err = 1
 
-src = {i:(100+i) for i in range(nhost)}
-print src
+src = [(i,(100+i)) for i in range(nhost)]
```

Comment:
The whole point of the test is that py_alltoall(dict) is not valid. But seems to be failing too severely (circumventing the 'try') when nhost > 1?  Perhaps should just begin with
```
# A nice improvement to pc.py_alltoall() is that it accept a dict as an arg.
if pc.nhost() > 1:
  if pc.id() == 0:
    print(0)
  pc.barrier()
  quit()
```

Reply:
this is failing in some `NEURON CI` pipelines: https://github.com/neuronsimulator/nrn/pull/1484/checks